### DISCUSSION TITLE
How can I use the List Generator?

### DISCUSSION THREAD

**Discussion by Aholicknight** [openbullet/openbullet#866] [Q&A] [answered]
![image](https://user-images.githubusercontent.com/7843719/195641335-12b4b425-327b-468e-91c6-ba1e32db28f4.png)

I am trying to generate a list of codes but when I click generate I get a bunch of zeros. Is there documentation what the mask is supposed to be like? Thank you!

**Comment by CryptoContact** [comment, marked as answer]
Change the 0s to *
